src/cloud/docker/docker-mode-developer.md

```diff
@@ -34,6 +34,9 @@ To launch the Docker environment in developer mode:
    ```bash
    ./vendor/bin/ece-docker build:compose --mode="developer"
    ```
+   {:.bs-callout-info}
+      This mode will be used in every further deployment by `docker-compose run --rm deploy cloud-deploy`, see step 6 .
```

Comment:
Two things wrong here. First, a missing line. Second, we cannot call out a step number, since that is assigned and not written. Also recommended some formatting and grammatical adjustment.

```suggestion

   {:.bs-callout-info}
   The mode specified in the compose command determines the deployment for the `docker-compose run --rm deploy cloud-deploy` command in a following step.
```